proxy/auth/auth.go

```
@@ -0,0 +1,176 @@
+
+func SetCookie(w http.ResponseWriter, value string) {
+	cookie := http.Cookie{
+		Name:     common.CookieSessionName,
+		Value:    value,
+		Secure:   config.TlsCertPath != "",
+		HttpOnly: true,
+		SameSite: http.SameSiteStrictMode,
+		Path:     "/",
+	}
+	http.SetCookie(w, &cookie)
+	w.Write([]byte("Session cookie set"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)